### PR TITLE
Switch 0.5 install instructions to use chart tags

### PR DESCRIPTION
**Pull request by scothis**

### CODE REVIEW

File: docs/v0.5/getting-started/docker-for-windows.md [hunk 1]
```diff
@@ -135,13 +135,13 @@ Install riff with both the Core and Knative runtimes. To omit or include other r
 
 ```powershell
 helm install projectriff/riff --name riff `
-  --set riff.runtimes.core.enabled=true `
-  --set riff.runtimes.knative.enabled=true `
-  --set riff.runtimes.streaming.enabled=false `
+  --set tags.core-runtime=true \
```
Comment (trisberg) [inline docs/v0.5/getting-started/docker-for-windows.md]: line continuation for windows should be ` and not \

**Review by trisberg — CHANGES_REQUESTED** [review]
Looks good except for the windows line continuation

File: docs/v0.5/getting-started/docker-for-windows.md (hunk 1 above)
Comment (scothis) [reply docs/v0.5/getting-started/docker-for-windows.md]: copy-paste

Comment (trisberg) [reply docs/v0.5/getting-started/docker-for-windows.md]: added a commit

**Review by trisberg — APPROVED** [review]

**Pull request merged**